internal/tfimport/importer/resources_test.go

```diff
@@ -199,23 +198,20 @@ func TestLoadFields(t *testing.T) {
 
 func TestLoadFieldsErr(t *testing.T) {
 	tests := []struct {
-		fields []string
+		fields  []string
+		wantErr error
```

Comment:
nit: I know I suggested this earlier but wantErr is a bit unncessary since this is the errors test case, just want is fine.

Reply:
Done.